### ISSUE TITLE
Remove example AccessControl.ino

### ISSUE THREAD

**Issue by Rotzbua** [miguelbalboa/rfid#568]
tl;dr: example has bug -> confuses people -> move to wiki

## Background
The example was added in 2016 and in terms of complexity and scope it was more of an example project than a simple use case for the library.

## Problem
Unfortunately, some problems have been revealed which have not been fixed yet. It seems that there are two opposing bugs, which means that if you fix one bug, the other bug will reappear. Therefore, previous bug fixes were not sufficient.

## Assessment
I think that there is a fundamental problem which can only be solved by a complete rework.

## Solution
Since an example that does not work as expected confuses new users rather than benefits them, the code is removed from the repository and added to the wiki.

## Reference

#403 #567

**Comment by Rotzbua** [comment]
Available in the wiki:
https://github.com/miguelbalboa/rfid/wiki/%5BExample%5D-AccessControl
